php/src/Docker/DockerHubManager.php

```diff
@@ -26,29 +26,30 @@ public function GetLatestDigestOfTag(string $name, string $tag) : ?string {
         // If one of the links below should ever become outdated, we can still upgrade the mastercontainer via the webinterface manually by opening '/api/docker/getwatchtower'
 
         try {
-            $authTokenRequest = $this->guzzleClient->request(
+            $request = $this->guzzleClient->request(
                 'GET',
-                'https://auth.docker.io/token?service=registry.docker.io&scope=repository:' . $name . ':pull'
+                'https://hub.docker.com/v2/repositories/' . $name . '/tags?page_size=128'
```

Comment:
@Zoey2936 , do you think requesting the most recent 128 tags is enough?

Reply:
I would say yes

Reply:
okay, thanks